keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -506,6 +509,30 @@ With recent conversations about kube-apiserver and enabled featureset version,
 it is possible to incorporate other version relating to the cluster, such as
 minimum kubelet version, maximum kubelet version, and enabled featureset version.
 
+#### Credentials
+
+An array of credentials to access the cluster. Each credential should be assigned 
+to a different consumer. Each credential has: 
+- a **consumer** field to indicate which consumer it is assigned to.
+- a **context** field to indicate the context name in the access information that should be used to access the cluster.
+- an **accessRef** field to point to the access information location. 
+
+The access information itself is stored in the same Kubernetes cluster on which
+the ClusterProfile API resides. However, the access information for different 
+consumers is stored in different namespaces to ensure that it is only accessible
+to the consumer for which it is intended. We recommend that the access information
+be stored in a secret object to add further protection to the access information.
```

Comment:
Should there recommendations on lifecycle of the credentials:
1/ Credentials must exist before being referred to in the cluster profile?
2/ Credentials or the context must be cleaned up after deletion of the cluster profile, what is the expectation?
3/ How are credentials rotated (is it necessary?)? Is there expectation on the consumer's side to regularly reload the credentials?